src/modules/sipcapture/sipcapture.c

```diff
@@ -434,6 +436,64 @@ struct module_exports exports = {
 	destroy          /* module destroy function */
 };
 
+/*
+ * Find the first occurrence of find in s, where the search is limited to the
+ * first slen characters of s.
+ */
+char *strnstr(const char *s, const char *find, size_t slen)
```

Comment:
Please try to use the existing implementation:
src/core/str.h
130:char* _strnstr(const char *s, const char *find, size_t slen);